app/src/main/res/layout/fragment_email_login.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
The FrameLayout is only necessary inside the activity layout, not the fragment layouts so if you could remove them from each of the fragment xml files that would be great!